config/defaults.yml

```diff
@@ -73,6 +73,7 @@ defaults:
     enable_registrations: true
     enable_local_posts_stream: 'disabled'
     autofollow_on_join: true
+    autofollow_on_join_accounts: ['hq@pod.diaspora.software']
     autofollow_on_join_user: 'hq@pod.diaspora.software'
```

Comment:
Likewise the code should probably just handle the setting being absent and thus the default should no longer be needed.